tests/yam/validate/validate_snapshot.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate that the snapshot "after install" is not present.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    select_console 'root-console';
+
+    my $snapper_description = script_output("snapper list --columns description");
```

Comment:
before running the command filtered we should show the snapper list to make the test more clear.